core/domain/user_validators_test.py

```diff
@@ -3019,7 +3033,8 @@ def test_missing_user_settings_model_failure(self):
             expected_output, sort=False, literal_eval=False)
 
     def test_user_settings_model_not_marked_deleted_failure(self):
-        user_model = user_models.UserSettingsModel.get_by_id(self.user_id)
+        user_model = user_models.UserSettingsModel.get_by_id(
```

Comment:
Should this be ``get()`` now?

Reply:
https://github.com/oppia/oppia/pull/11914#discussion_r578965289

Reply:
Resolving this as this is being discussed in the above conversation.